### PR TITLE
fix: wrong ref for global variables

### PR DESCRIPTION
**Pull request by tadelesh**
- [ ] The purpose of this PR is explained in this or a referenced issue.
- [ ] The PR does not update generated files.
   - These files are managed by the codegen framework at [Azure/autorest.go][].
- [ ] Tests are included and/or updated for code changes.
- [ ] Updates to [CHANGELOG.md][] are included.
- [ ] MIT license headers are included in each file.

[Azure/autorest.go]: https://github.com/Azure/autorest.go
[CHANGELOG.md]: https://github.com/Azure/azure-sdk-for-go/blob/main/CHANGELOG.md

### CODE REVIEW

**Review by Alancere — APPROVED** [review]

File: azure-pipelines.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
       name: $(pool.name)
 
     variables:
-      - template: ../variables/globals.yml
+      - template: ./eng/pipelines/templates/variables/globals.yml
```
Comment (weshaggard) [inline azure-pipelines.yml:32]: FYI @benbp looks like this was from your earlier PR. We should make this an absolute path `/eng/pipelines/templates/variables/globals.yml`

Comment (benbp) [reply azure-pipelines.yml:32]: Not sure how this passed CI at the time. https://github.com/Azure/azure-sdk-for-go/pull/17756

**Pull request merged**